python/mxnet/symbol.py

```diff
@@ -12,7 +12,7 @@
 from .base import _LIB
 from .base import c_array, c_str, mx_uint, py_str, string_types, mx_real_t
 from .base import NDArrayHandle, ExecutorHandle, SymbolHandle
-from .base import check_call, ctypes2docstring
+from .base import check_call, ctypes2docstring, MXNetError
```

Comment:
The mxneterror thing doesn't seem very infromative. How about use appropriate default errors?